decidim-admin/app/controllers/decidim/admin/application_controller.rb

```diff
@@ -8,6 +8,7 @@ class ApplicationController < ::DecidimController
       include NeedsPermission
       include NeedsPasswordChange
       include NeedsSnippets
+      include NeedsAdminTosAccepted
       include FormFactory
       include LocaleSwitcher
       include UseOrganizationTimeZone
```

Comment:
Unrelated to this PR directly but I also noticed that we are redirecting normal users also to the admin root path in this controller in case they try to access some admin view.

I think the `user_has_no_permission_path` method in this controller should look like this:
```ruby
      def user_has_no_permission_path
        return decidim.root_path if user_signed_in? && !current_user.admin?

        decidim_admin.root_path
      end
```